Applications/FileIO/GocadIO/GocadAsciiReader.h

```diff
@@ -26,31 +26,44 @@ namespace FileIO
 {
 namespace Gocad
 {
-class GocadTSurfaceReader final
+
+enum class GOCAD_DATA_TYPE
+{
+    UNDEFINED,
+    VSET,
+    PLINE,
+    TSURF,
+    MODEL3D,
+    ALL
+};
+
+class GocadAsciiReader final
 {
 public:
     /**
      * Constructor takes as argument the Gocad .sg text file.
```

Comment:
Wrong comment. Default ctor does not take any arguments.

Reply:
:heavy_check_mark: